### PR TITLE
Switch SES API to Signature v4

### PR DESCRIPTION
**Pull request by gproskurin**
Fixes https://github.com/erlcloud/erlcloud/issues/652

### CODE REVIEW

File: src/erlcloud_ses.erl [hunk 1]
```diff
@@ -1242,38 +1242,10 @@ decode_error(Doc) ->
 %%%------------------------------------------------------------------------------
 
 ses_request(Config, Action, Params) ->
-    case erlcloud_aws:update_config(Config) of
```
Comment (nalundgaard) [inline src/erlcloud_ses.erl:1245]: Please don't delete this.

Comment (gproskurin) [reply src/erlcloud_ses.erl:1245]: You mean, keep call to `update_config`? It is called in `aws_request4`

https://github.com/erlcloud/erlcloud/blob/0f0540041ad6b55700715d1d3629853bf3158791/src/erlcloud_aws.erl#L184


And this is similar to what SQS code does:
https://github.com/erlcloud/erlcloud/blob/0f0540041ad6b55700715d1d3629853bf3158791/src/erlcloud_sqs.erl#L677-L678

Comment (nalundgaard) [reply src/erlcloud_ses.erl:1245]: Oh, nice! I didn't realize that. Thanks

**Comment by gproskurin** [comment]
Please see links in discussion above. `update_config` is done in `aws_request4`

**Pull request merged**